modules/openxr/doc_classes/OpenXRInterface.xml

```diff
@@ -10,6 +10,27 @@
 	<tutorials>
 		<link title="Setting up XR">$DOCS_URL/tutorials/xr/setting_up_xr.html</link>
 	</tutorials>
+	<methods>
+		<method name="get_available_display_refresh_rates" qualifiers="const">
+			<return type="Array" />
+			<description>
+				Returns display refresh rates supported by the current HMD. Only returned if this feature is supported by the OpenXR runtime and after the interface has been initialised.
+			</description>
+		</method>
+		<method name="get_display_refresh_rate" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the current display refresh rate for the current HMD. Only returned if this feature is supported by the OpenXR runtime and after the interface has been initialised.
+			</description>
+		</method>
+		<method name="set_display_refresh_rate">
+			<return type="void" />
+			<param index="0" name="refresh_rate" type="float" />
+			<description>
+				Set the display refresh rate for the current HMD, must be a rate returned by [method get_available_display_refresh_rates]. Only applied if this feature is supported by the OpenXR runtime and after the interface has been initialised.
+			</description>
+		</method>
```

Comment:
Should it be a property?

Reply:
The problem I have with a property is that it will try and set it at construction of the class while in this case the display rate is defunct until the interface is fully setup.

That said, seeing how these objects are constructed it may not really be an issue.

Reply:
Yeah seeing the code it actually makes sense to keep these as methods only I think, it's not a typically setter/getter pattern.